### PR TITLE
gildas: add alternative url for the source code

### PR DESCRIPTION
**Pull request by smaret**
###### Motivation for this change

For each new release, the upstream developers of Gildas move the source code of the previous release to a different directory. Add this directory to the list of url to avoid build failure at each new
release.

Please cherry-pick to 18.09 when merging.

/cc #45960

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
@GrahamcOfBorg build gildas

File: pkgs/applications/science/astronomy/gildas/default.nix [hunk 1]
```diff
@@ -12,7 +12,8 @@ stdenv.mkDerivation rec {
   name = "gildas-${version}";
 
   src = fetchurl {
-    url = "http://www.iram.fr/~gildas/dist/gildas-src-${srcVersion}.tar.gz";
+    urls = [ "http://www.iram.fr/~gildas/dist/gildas-src-${srcVersion}.tar.gz"
```
Comment (Mic92) [inline pkgs/applications/science/astronomy/gildas/default.nix]: Please also add a comment like:

```nix
# For each new release, the upstream developers of Gildas move the source code of the previous release to a different directory
```

Comment (smaret) [reply pkgs/applications/science/astronomy/gildas/default.nix]: Done with 22fb8ca

**Comment by Mic92** [comment]
[detached HEAD 0ba0f158efc] gildas: add alternative url for the source code (#46145)
 Author: Sebastien Maret <sebastien.maret@icloud.com>
 Date: Thu Sep 6 16:10:21 2018 +0200
 1 file changed, 4 insertions(+), 1 deletion(-)

**Comment by smaret** [comment]
Thanks @Mic92

**Pull request merged**